cmd/istioctl/main.go

```diff
@@ -38,20 +52,30 @@ var (
 	schema model.ProtoSchema
 
 	postCmd = &cobra.Command{
-		Use:   "create [type] [name]",
-		Short: "Create a configuration object",
+		Use:   "create",
+		Short: "Create configuration objects",
 		RunE: func(c *cobra.Command, args []string) error {
-			if len(args) != 2 {
-				return fmt.Errorf("provide configuration type and name")
+			if len(args) != 0 {
+				return fmt.Errorf("create takes no arguments")
 			}
-			if err := setup(args[0], args[1]); err != nil {
-				return err
-			}
-			v, err := readInput()
+			// Initialize schema global
+			varr, err := readInputs()
 			if err != nil {
 				return err
 			}
-			return cmd.Client.Post(key, v)
+			if len(varr) == 0 {
+				return errors.New("Nothing to create")
+			}
+			for _, v := range varr {
+				setup(v.Type, v.Name)
+				err = cmd.Client.Post(key, v.InputDoc)
```

Comment:
See previous comment about combining assignment and check of `err` into single line.